src/api/tests/participantsService.spec.ts

```diff
@@ -1,90 +1,112 @@
-import request, { Request } from 'supertest';
-
-import { ParticipantRequest } from '../services/participantsService';
-import { mockParticipant, mockUser, mockUserOnce } from './queryMocks';
-import useTestServer, { api, routers } from './utils';
+import { NextFunction, Response } from 'express';
+import { Knex } from 'knex';
+
+import { TestConfigure } from '../../database/TestSelfServeDatabase';
+import {
+  createParticipant,
+  createResponseObject,
+  createUser,
+} from '../../testHelpers/apiTestHelpers';
+import { checkParticipantId, ParticipantRequest } from '../services/participantsService';
+
+const createParticipantRequest = (
+  email: string,
+  participantId: string | number
+): ParticipantRequest => {
+  return {
+    auth: {
+      payload: {
+        email,
+      },
+    },
+    params: {
+      participantId,
+    },
+  } as unknown as ParticipantRequest;
```

Comment:
I don't love this but I couldn't figure out a better way to do this. Open to ideas

Reply:
I think it's OK in this case, especially because it's a test.